### ISSUE TITLE
[Documentation] Clearly call out the port that rke2 server is listening on for agent registrations

### ISSUE THREAD

**Issue by dnoland1** [rancher/rke2#165]
Installed server (10.0.7.10) using:
```
INSTALL_RKE2_VERSION=v1.18.4-beta1+rke2 ./install.sh
```
Looking good:
```
# kubectl get nodes
NAME           STATUS   ROLES         AGE     VERSION
ip-10-0-7-10   Ready    etcd,master   3h15m   v1.18.4-beta1+rke2
```
Grabbed a copy of the token at `/var/lib/rancher/rke2/server/node-token`

Installed agent (10.0.7.11) using:
```
INSTALL_RKE2_VERSION=v1.18.4-beta1+rke2 RKE2_URL=https://10.0.7.10:6443 RKE2_TOKEN=K1029a02c57fb20b76f2f026adfb122565f9f971db76ebe002acd85fc6b76fc09b7::server:7297b29e32acaacbf1c16e698c078f8e ./install.sh
```
Isn't showing up in kubectl get nodes. Seeing this error every 2 seconds in the logs:
```
Aug 07 18:00:14 ip-10-0-7-11 rke2[1470]: time="2020-08-07T18:00:14Z" level=error msg="failed to get CA certs at https://127.0.0.1:38853/cacerts: https://127.0.0.1:38853/cacerts: 401 Unauthorized"
Aug 07 18:00:16 ip-10-0-7-11 rke2[1470]: time="2020-08-07T18:00:16Z" level=error msg="failed to get CA certs at https://127.0.0.1:38853/cacerts: https://127.0.0.1:38853/cacerts: 401 Unauthorized"
```

**Comment by briandowns** [comment]
Can you change the port from 6443 to 9432 and see what happens?

**Comment by dnoland1** [comment]
@briandowns I get a different error:
```
Aug 07 20:02:38 ip-10-0-7-11 rke2[2009]: time="2020-08-07T20:02:38Z" level=info msg="Starting rke2 agent v1.18.4-beta1-rke2 (HEAD)"
Aug 07 20:02:38 ip-10-0-7-11 rke2[2009]: time="2020-08-07T20:02:38Z" level=info msg="module overlay was already loaded"
Aug 07 20:02:38 ip-10-0-7-11 rke2[2009]: time="2020-08-07T20:02:38Z" level=info msg="module br_netfilter was already loaded"
Aug 07 20:02:38 ip-10-0-7-11 rke2[2009]: time="2020-08-07T20:02:38Z" level=info msg="Running load balancer 127.0.0.1:33047 -> [10.0.7.10:9432]"
Aug 07 20:02:38 ip-10-0-7-11 rke2[2009]: time="2020-08-07T20:02:38Z" level=error msg="failed to get CA certs at https://127.0.0.1:33047/cacerts: Get \"https://127.0.0.1:33047
/cacerts\": read tcp 127.0.0.1:45236->127.0.0.1:33047: read: connection reset by peer"
```
Last line repeats every 2 seconds.

I checked the server and didn't see anything listening on port 9432. However, I did see that `rke2` was listening on port 9345. I tried that port and that works:
```
root@ip-10-0-7-10:~# kubectl get nodes
NAME           STATUS   ROLES         AGE     VERSION
ip-10-0-7-10   Ready    etcd,master   8m36s   v1.18.4-beta1+rke2
ip-10-0-7-11   Ready    <none>        102s    v1.18.4-beta1+rke2
```
Looks like the README.md needs to be updated.

**Comment by briandowns** [comment]
Ugh, my apologies. Try again with port 9345.

**Comment by cjellick** [comment]
Yes, the solution is port 9345

**Comment by brandond** [comment]
Fixed in #337

**Comment by einsteinarbert** [comment]
in my case: I just change `/etc/rancher/rke2/config.yaml` domain to ip address of `9345` port
